scripts/build_aggregate.py

```diff
@@ -17,12 +33,24 @@ def compile_single_corpus(corpus_name: str) -> pd.DataFrame:
     df.insert(0, "corpus_version", corpus_string)

```

Comment:
Update this column name "corpus_version" to "corpus" (keep the version string attached to the corpus name though). Update the docstring accordingly